Cargo.toml

```diff
@@ -96,8 +89,8 @@ zipf = "5.0.1"
 bitflags = "1.0.1"
 fail = "0.3"
 uuid = { version = "0.6", features = [ "serde", "v4" ] }
-grpcio = { version = "0.5.0-alpha.3", features = [ "openssl-vendored" ] }
-raft = "0.6.0-alpha"
+grpcio = { version = "0.5.0-alpha.3", features = [ "openssl-vendored" ], default-features = false }
```

Comment:
why disable default features here and following deps?

Reply:
Because when we use grpc-rs etc., we must use either prost-codec or protobuf-codec (which is the default), but not both.

Reply:
got it